lib/ui/src/components/notifications/item.tsx

```diff
@@ -1,34 +1,141 @@
-import React, { FunctionComponent } from 'react';
+import React, { FunctionComponent, SyntheticEvent } from 'react';
```

Comment:
Rename this component and filename NotificationItem

Reply:
The best app